spec.html

```diff
@@ -313,9 +313,7 @@
       <h1>BigInt::leftShift (_x_, _y_)</h1>
       <p>The abstract operation BigInt::leftShift with two arguments _x_ and _y_ of BigInt:</p>
       <emu-alg>
-        1. If _y_ &lt; 0,
-          1. Return a BigInt representing _x_ divided by 2<sup>_y_</sup>, rounding down to the nearest integer, including for negative numbers.
```

Comment:
I'd actually like to keep the original, more wordy phrasing, since I believe we'll work towards a specification with no mathematical values and instead all numeric values in spec-land being either Numbers or BigInts (see #10). The small fraction in your spec text is not always representable as either. We could fix the issue more minimally by just inserting a `-` here, right?